gcc/config/riscv/riscv.cc

```diff
@@ -10859,7 +10859,7 @@ riscv_sched_reorder2 (FILE *file ATTRIBUTE_UNUSED,
 	  if (NONDEBUG_INSN_P (ready[*n_readyp - i])
 	      && !SCHED_GROUP_P (ready[*n_readyp - i])
 	      && (!next_insn (ready[*n_readyp - i])
-		  || !NONDEBUG_INSN_P (next_insn (ready[*n_readyp - i]))
+		 // || !NONDEBUG_INSN_P (next_insn (ready[*n_readyp - i]))
```

Comment:
Don't just comment these out. Remove the lines if they need to be removed.
Also, don't we need something like `next_non_dbg_insn`? That keeps invoking `next_insn` in a loop until it finds either a non-debug insn or null?